package-lock.json

```diff
@@ -22,7 +22,7 @@
         "jsonwebtoken": "^8.2.0",
         "lodash": "^4.17.21",
         "morgan": "^1.10.0",
-        "orientjs": "^3.0.11",
+        "orientjs": "^3.2.0",
```

Comment:
@sshu, do these changes to the package-lock.json file has been made manually or by running npm install?

Reply:
yes

Reply:
yes what? lol. manually?

Reply:
sorry, I mean it is made automatically by running npm install. This change came from https://github.com/bcgsc/pori_graphkb_api/pull/68

Reply:
ok ty!